scenarios/scenario-10/README.md

```diff
@@ -0,0 +1,116 @@
+## Scenario 10 - Apply interceptors to the API
+- This scenario describes how to apply interceptors to carry out transformations and mediations on the requests and responses.
+- First, we need to implement custom request interceptors and response interceptors. For more information about implementing custom interceptors you can refer the document [adding interceptors.](https://docs.wso2.com/display/MG300/Adding+Interceptors)
+- Then we need to Initialize a new WSO2 API Manager project and add the custom interceptor files in interceptors folder.
+- We need to refer the interceptors in swagger definition in order to apply them on the requests and responses.
+- Finally, we will invoke the API and observe how the added interceptors act on requests and responses.
+
+ ***Important:***
+> Follow the main README and deploy the apim-operator and configuration files. Make sure to set the analyticsEnabled to "true" and deploy analytics secret with credentials to analytics server and certificate, if you want to check analytics.
+
+
+ ##### Deploying the artifacts
+ 
+ - Init the API project using CLI. This will Initialize a new WSO2 API Manager project in same directory.
+     ```
+        apictl init petstore-int --oas=./scenarios/scenario-10/interceptor_swagger.yaml
+     ```
+     - Output:
+     ```      
+        Initializing a new WSO2 API Manager project in ./product-apim-tooling/import-export-cli/build/target/apimcli/petstore-int
+        Project initialized
+        Open README file to learn more
+     ```
+  
+ - Copy the _modifyResponse.bal_ and _validateQueryPrams.bal_ files in ./scenarios/scenario-10/ into the interceptors folder in ./product-apim-tooling/import-export-cli/build/target/apimcli/petstore-int path.
```

Comment:
Avoid using camelCase and hypen-case both in a single name. Use a single convention.

Reply:
There are multiple places. please check :)

Reply:
my bad @Shehanir . :D Ignore the above two comments.